cacheback/base.py

```diff
@@ -353,7 +353,9 @@ def hash(self, value):
 
         This is for use in a cache key.
         """
-        return hashlib.md5(to_bytestring(value)).hexdigest()
+        if isinstance(value, tuple):
+            value = tuple(to_bytestring(v) for v in value)
+        return hashlib.md5(six.b(':').join(value)).hexdigest()
```

Comment:
So this basically goes through are parameters to make sure the stuff that is turned into a hash actually is what we expect. The conversion to bytestring in `to_bytestring` was actually never really applied